src/OrchardCore.Modules/OrchardCore.OpenId/Views/Application/Edit.cshtml

```diff
@@ -49,7 +49,7 @@
         <div class="custom-control custom-checkbox">
             <input type="checkbox" class="custom-control-input" asp-for="AllowImplicitFlow" data-toggle="collapse" data-target="#AllowImplicitFlowRecommendedHint" checked="@Model.AllowImplicitFlow">
             <label class="custom-control-label" asp-for="AllowImplicitFlow">@T["Allow Implicit Flow"]</label>
-            <span class="hint">@T["— {0}", "More info:"] <a href="http://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth">http://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth</a></span>
+            <span class="hint">— @T["More info:"] <a href="http://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth">http://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth</a></span>
```

Comment:
@hishamco how will this kind of thing work when localized in RTL ? will the link and text and dash work correctly?

Reply:
It should work without any issue, coz the page direction will control the flow

> will the link and text and dash work correctly?

Yes, if all the text after and before are translated